lldb/source/Plugins/Process/gdb-remote/GDBRemoteCommunicationServerPlatform.cpp

```diff
@@ -159,6 +159,40 @@ GDBRemoteCommunicationServerPlatform::GDBRemoteCommunicationServerPlatform(
 GDBRemoteCommunicationServerPlatform::~GDBRemoteCommunicationServerPlatform() =
     default;
 
+void GDBRemoteCommunicationServerPlatform::Proc(
```

Comment:
If we're changing the name could we have a better name than `Proc`, or if `Proc` is just a term I am unfamiliar with, a comment in the header to explain what this function does?

Reply:
Note `void SetInferiorArguments(const lldb_private::Args &args);` is a missing function. It is a bug. I have just removed it and added Proc(). There is no any connection between these functions. I have just moved a common code from lldb-platform to GDBRemoteCommunicationServerPlatform::Proc(). We can rename it to ClientHandle() or such and even kept inside lldb-platform.cpp